Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.MixedReality.Toolkit.Utilities;
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace Microsoft.MixedReality.Toolkit.Input
```

Comment:
>MixedReality [](start = 20, length = 12)

Can you also add a link/mention on this issue:

[https://github.com/microsoft/MixedRealityToolkit-Unity/issues/5772](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/5772)

And also a new section here:

[https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Updating.html](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Updating.html)

To describe the changes here (i.e. deletion hand yaw/rotate controls and how to use that functionality in the new world?)

Reply:
It's cheaper for us to do this as we go along the month than trying to write this all at the end (and potentially forgetting some stuff as we go)

---
In reply to: [320507000](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5853#discussion_r320507000) [](ancestors = 320507000)

Reply:
done!